extensions/amp-bind/0.1/bind-validator.js

```diff
@@ -36,6 +36,9 @@ const GLOBAL_PROPERTY_RULES = {
   'class': {
     denylistedValueRegex: '(^|\\W)i-amphtml-',
   },
+  'class+': {
+    denylistedValueRegex: '(^|\\W)i-amphtml-',
```

Comment:
Self review, these literals are the same. Consider if there is a better pattern.